src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -303,7 +280,7 @@ class DefaultColorControlSceneHandler : public scenes::DefaultSceneHandlerImpl
         // Switch to the mode saved in the scene
         if (SupportsColorMode(endpoint, targetColorMode))
         {
-            ColorControlServer::Instance().handleModeSwitch(endpoint, targetColorMode);
+            ColorControlServer::Instance().handleModeSwitch(endpoint, static_cast<uint8_t>(targetColorMode));
```

Comment:
```suggestion
            ColorControlServer::Instance().handleModeSwitch(endpoint, to_underlying(targetColorMode));
```
Again: please do not use static_cast to convert from enums to integers.  That loses size information that can catch bugs.